tests/Aspire.Milvus.Client.Tests/ConformanceTests.cs

```diff
@@ -0,0 +1,100 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using Aspire.Components.ConformanceTests;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Milvus.Client;
+using Xunit;
+
+namespace Aspire.Milvus.Client.Tests;
+
+public class ConformanceTests : ConformanceTests<MilvusClient, MilvusClientSettings>
+{
+    protected override bool SupportsKeyedRegistrations => true;
+
+    protected override bool CanCreateClientWithoutConnectingToServer => false;
+
+    protected override bool CanConnectToServer => RequiresDockerTheoryAttribute.IsSupported;
+
+    protected override ServiceLifetime ServiceLifetime => ServiceLifetime.Singleton;
+
+    protected override string[] RequiredLogCategories => Array.Empty<string>();
+
+    protected override string ActivitySourceName => "";
+
+    protected override void RegisterComponent(HostApplicationBuilder builder, Action<MilvusClientSettings>? configure = null, string? key = null)
+    {
+        if (key is null)
+        {
+            builder.AddMilvusClient("milvus", configure);
+        }
+        else
+        {
+            builder.AddKeyedMilvusClient(key, configure);
+        }
+    }
+
+    protected override void PopulateConfiguration(ConfigurationManager configuration, string? key = null)
+        => configuration.AddInMemoryCollection(new KeyValuePair<string, string?>[2]
+        {
+            new KeyValuePair<string, string?>(CreateConfigKey("Aspire:Milvus:Client", key, "Endpoint"), "http://localhost:19530"),
+            new KeyValuePair<string, string?>($"ConnectionStrings:{key}","Endpoint=http://localhost:19530;Key=pass")
+        });
+
+    protected override void TriggerActivity(MilvusClient service)
+    {
+        service.GetVersionAsync().Wait();
+    }
```

Comment:
Do we have any tracing with Milvus?

What is the tracing, metrics, and health checks plan for this component?

Reply:
I was planning on logging issues on the component to add some of these. @roji do you know if these may already be in the works?

Reply:
The tracking issue on the Milvus SDK side is https://github.com/milvus-io/milvus-sdk-csharp/issues/85.

I'm planning on adding tracing support there, but it may take me a bit of time before I get around to it... Re metrics, I **think** it's more appropriate to get them from Milvus itself (the server), which emits them - see the issue above for some discussion on that. We may want to have a more general discussion on the client vs. server preference for metrics.